lib/rules/max-empty-lines/index.js

```diff
@@ -207,7 +207,7 @@ function isEofNode(document, root) {
 		after = _.get(document.nodes[rootIndex + 1], 'raws.beforeStart');
 	}
 
-	return !(after + '').trim();
+	return !`${after}`.trim();
```

Comment:
```suggestion
	return !String(after).trim();
```

What do you think? I believe it's easier to read than template literals.

Reply:
@hudochenkov I had already a branch for this stuff which I plan to tweak the ESLint rules and will help us with such cases https://github.com/stylelint/stylelint/compare/master-xmr-eslint

So, I'd say this should land with only the template literal changes.